### ISSUE TITLE
I am not trying to be game developer

### ISSUE THREAD

**Issue by davidwesst** [davidwesst/blog#6]
Hey friendos, it's me DW. 

It's been a while and I'm still working on making my game, Out the Door. It's been a journey, both on the tech side and for myself, but this post isnt about the game directly. 

This one is about me, and the truth that I've come up realize since my last video: 
I am not trying to be a game developer...but I am definitely trying to make a game. 

# What does that even mean? 
It means that I'm not aiming to get into game development business or quit my job or anything oh so drastic. 

I'm in it for myself and to achieve a life goal: to make and sell a video game of my own design on PC and a console.

But, I'm not trying to change my career. If my game ends up some runaway indie success story, sure I'll consider it, but that's not the goal. The goal is more personal than professional, and I think that distinction is important for you to know. 

# Uh...cool. I mean't what does mean _for me, the viewer_

*cough* Of course...

It means my content is going to vary a bit. YouTube content is still going to focus around game development, but expect to see some content around what is currently influencing my game. making or just reviews and analysis of tools I'm exploring. 

You know, when I write/say it out loud, it's not that different than it has been the past couple of years. 

Really, it's just more of a mind shift for me. I'm streaming gameplay gameplay over on Twitch, where I'm going through my personal collection of games and reviving the WTF series.

I've also revived my blog, where I share my inner monologue, whatever that may be that week. It ranges beyond game development, but if you want to know more about my mindset and work, that's where you'll find more. 

# TL;DR; / Conclusion
I'm finally ready to return to YouTube with gamedev content, but my mindset is different. The truth is, I'm not trying to be a game developer, but rather I'm trying to achieve a life goal my making a game. 

My blog is back with my full inner monologue, and I'm searching for inspiration in my personal collection over on Twitch.

YouTube will stay game dev focused, but expect to hear more about what I find and learn on my journey, rather than just project updates.

Thanks for playing. 

I am trying make a game. There is a huge difference it turns out.

- Those statements sound contradictory but the differences in my interpretation make a difference.
- Not trying to change my career into coding and making games. If it happens and the opportunity arises, then awesome. But that is like trying to win the lottery with the effort I am able (and willing to put in).
- But...I am trying to make a game. It's the dream I have had since I was a kid and now that I'm old, have grown up responsibilities, and two kids I finally realized that these dreams COULD be a thing I actually do.
- So...youre a big phoney? 
Sometimes it felt like that, but I don't think so. 

- Why does it matter? 
It matters to me because the way I have been thinking about gamedev, is the same way that I have been thinking about my professional development for my day job (solution architect in higher education). But the goals are different, as the day job is geared toward career advancement while gamedev is geared towards personal growth and accomplishment. 
- What does it all mean? 
Just that I should be more honest with myself when it comes to effort in projects and content creations. My journey with game development isn't the same as when I was in the MVP Program and consulting in Web Platform technology. This journey is for me and hopefully others in a similar position.
